static/js/main.js

```diff
@@ -51,6 +51,30 @@ $(() => {
     return false;
   });
 
+  // Allow reordering identities
+  $("form.identities").sortable({
+    items: ".identity",
+    axis: "y",
+    containment: "parent",
+    deactivate: evt => {
+      var identities = $(evt.target);
+      var url = '/api/identities/reorder/';
+      var cards = identities.find('.identity');
+      var data = $.map(cards, card => $(card).data('id'));
+      var csrftoken = getCookie('csrftoken');
+
+      fetch(url, {
+        method: "POST",
+        body: JSON.stringify(data),
+        credentials: "same-origin",
+        headers:{
+          "Content-Type": "application/json",
+          "X-CSRFToken": csrftoken
+        }
+      });
```

Comment:
Probably wise, yeah. It's not ideal (the UX is that the items snap back without explanation, possibly after you've started another interaction) but it's better than silently doing nothing.

Reply:
And while we're at that, we could throw a little "you had an error" toast, perhaps.

Reply:
I'm inclined to punt on toasts in favour of a ticket - we should do all the API call error handling at once, rather than doing it on a per-pull-request basis, I think, now that we _have_ this problem. Is that okay with you?

Reply:
100% yes.

Reply:
> Handle an error response by reverting order?
 
Done in 7e23ca9.